content/es/working-groups.md

```diff
@@ -0,0 +1,279 @@
+
+Los GT se reúnen semanalmente en un Google Hangout On Air. Un moderador designado aprobado por el GT lleva a cabo la reunión. Cada reunión debe ser publicada en YouTube.
+
+Se añaden items a la agenda del GT que sean considerados contenciosos o sean modificaciones en el gobierno, política de contribución, membresía del GT, o proceso de publicación.
+
+La intención de la agenda no es la de aprobar o revisar todos los parches; lo que ha de ocurrir de forma continua en GitHub y ha de ser llevada a cabo por un grupo más grande de Colaboradores.
+
+Cualquier miembro de la comunidad o colaborador puede preguntar si algo puede ser añadido a la agenda de la siguiente reunión abriendo un GitHub Issue. Cualquier colaborador, miembro de GT o moderador puede añadir un item a la agenda añadiendo la etiqueta **Agenda del GT** al issue.
+
+Con anterioridad a cada reunión del GT el moderador compartirá la Agenda con los miembros del GT. Los miembros del GT pueden añadir tantos items como deseen al principio de cada reunión. El moderador y el GT no pueden vetar o suprimir items.
+
+El GT puede invitar personas o representantes a ciertos proyectos para participar en capacidad de no votante.
+
+El moderador es responsable de resumir la discusión de cada item en la agenda y enviar una pull request después de la reunión.
+
+### Proceso de Búsqueda de Consenso
+
+El GT sigue un proceso de [Búsqueda de Consenso](http://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+a la hora de tomar decisiones.
+
+Cuando un item en la agenda aparece para alcanzar consenso el moderador preguntará "¿Alguien objeta?" como llamada final para disentir del consenso.
+
+Si un item en la agenda no alcanza consenso un miembro del GT puede hacer una llamada voto o a cerrar el voto para llevar este asunto a la mesa en la siguiente reunión. La llamada a voto debe de ser secundada por la mayoría del GT o de lo contrario la discusión continuará. Mayoría simple gana.
+
+Nótese que cambias a la membresía del GT requieren de un consenso unánime. Véase "Reuniones del GT" más arriba.
+
+### Certificado del desarrollador de Origen 1.0
+
+Contribuyendo a este proyecto, yo certifico que:
+
+* (a) La contribución fue creada en su todo o en parte por mi y que tengo el derecho de presentarla bajo una licencia de código indicada en el archivo; o
+* (b) La contribución está basada en un trabajo previo que, en el mejor de mi conocimiento, está cubierto bajo una licencia de código abierto apropiada y que yo tengo el derecho bajo esta licencia a presentar el trabajo con modificaciones, ya sea creado en su totalidad o en parte por mi bajo la misma licencia de código abierto (, is covered under an appropriate open source license
+  and I have the right under that license to submit that work with
+  modifications, whether created in whole or in part by me, under the
+  same open source license (a menos que me esté permitido el presentarlo bajo una licencia diferente), como está indicado en el archivo; o
```

Comment:
Auch! Thanks a lot. I keep the original text and write before it so I can always go back and forth between translation and original text.